templates/osu!.app/Contents/Info.plist

```diff
@@ -61,6 +61,22 @@
 			<key>LSTypeIsPackage</key>
 			<false/>
 		</dict>
+		<dict>
+			<key>CFBundleTypeName</key>
+			<string>osu! beatmap archive</string>
+			<key>LSHandlerRank</key>
+			<string>Default</string>
+			<key>CFBundleTypeExtensions</key>
+			<array>
+				<string>olz</string>
```

Comment:
Much of a muchness, but i do think this can just go in the extensions array above (alongside `osz`) rather than duplicating, since nothing else if changing.